task_5.1.py

```diff
@@ -0,0 +1,23 @@
+# Multi-paradigm programming languages, task №5.1
+# Andrey Fortus IKM-221a
+
+GENERAL_INFO = 'Multi-paradigm programming languages, task №5.1'
+STUDENT_INFO = 'Andrey Fortus IKM-221a variant №20'
+
+# info
+print(GENERAL_INFO)
+print(STUDENT_INFO)
+
+
+# function for finding the sum of a series
+def sum_series(n):
```

Comment:
move your functions to the top.
you can place it right after your copyright comment.